library/base/src/main/java/org/cru/godtools/base/Settings.kt

```diff
@@ -86,6 +92,9 @@ class Settings @Inject internal constructor(
         prefs.getStringLiveData(PREF_PARALLEL_LANGUAGE, null).distinctUntilChanged()
             .map { it?.toLocale() }
     }
+    val parallelLanguageFlow
+        get() = prefs.getStringFlow(PREF_PARALLEL_LANGUAGE, null).distinctUntilChanged()
```

Comment:
the ```getStringFlow``` here has ```null``` as the second value because it isn't required to have one set right? While the ```getStringFlow``` for the primary language is passed a default language because one has to be set no matter what.

Reply:
correct, `primaryLanguage` will always be non-null, `parallelLanguage` can be null